package.json

```diff
@@ -20,14 +20,14 @@
     ]
   },
   "dependencies": {
-    "@hapi/hoek": "^9.0.0"
+    "@hapi/hoek": "^10.0.0"
   },
   "devDependencies": {
-    "@hapi/code": "8.x.x",
+    "@hapi/code": "9.x.x",
```

Comment:
Since we're updating the version, should we move it to `npm`'s default range management aka caret?

Reply:
Sure!
```suggestion
    "@hapi/code": "^9.0.0",
```